### ISSUE TITLE
#Bug : Which-keys not showing up.

### ISSUE THREAD

**Issue by sinekonata** [darkstego/wakib.d#9]
Whenever I use `C-d-` (which was remapped from `C-c-` , I belive), I get this message from which-key :
`C-d-  which-key: There are no keys to show`
And no menu to see which keys. But at least in `org-mode` I know there are keys that are functional despite not being shown. Such as :
`C-d- '` or `C-d- *`
And this guy has this :
![Fri  5 Jun 23:54:32 CEST 2020](https://user-images.githubusercontent.com/2728722/83925652-f9b47c00-a787-11ea-8714-ef25ac85c599.png)
Is that an alternative to which-key?

How does one even see which keys are available in vanilla emacs anyway? Are you supposed to know them all? Thank you :D

**Comment by sinekonata** [comment]
Also I don't get that problem with `C-e-` (remapped from `C-x-`). Everything works normally for that key.

**Comment by darkstego** [comment]
In order to see which keys are available in vanilla Emacs you type Alt-x describe-bindings. It will give you a list of all the shortcuts. And the behavior you see is a bug in Emacs describe bindings.

There are actually two issues at play here. When you pass C-d as a key to describe bindings and pass another buffer (which-key uses a temp-buffer to get the bindings), the function suddenly returns nothing. I did fix this problem with the latest wakib-key. You can update by doing (Alt-x list-packages) Then Typing U to mark all packages for upgrade. Then x to execute.

The second problem is even in the same buffer, describe-key only shows some of the bindings  bindings in the same way. So depending on the minor mode you might only see a small fraction of the possible. This is a bug within describe-bindings which is a c-function in Emacs so there is no way to get around it. Strangely enough this only happens when you rebind the C-c key, which is why when you type C-e you see everything.

Another thing to keep in mind is that even in vanilla Emacs, which-key doesn't show all bindings because some prefixes aren't explicitly stated and thus don't show up. But it is a really convenient tool and I wish I was able to to get more bindings to show up.

Also, people don't usually know all the bindings. If you are talking about  C-d it mainly houses the major mode keybindings, it isn't usually that populated unless you are using org-mode, which has just a ton of bindings. If you are interested in org-mode bindings then I suggest finding the command in the toolbar and the shortcuts are shown there. They are so many that they need to be broken down into categories to find anything. I actually have a printed org-mode reference card placed on the wall above my monitor just to be able to find these.

TLDR: update wakib-key package and you will get some C-d bindings, but unfortunately an Emacs bug prevents them all from showing.
